projects/demo/project.json

```diff
@@ -68,6 +68,30 @@
                         }
                     ]
                 },
+                "github": {
+                    "baseHref": "/maskito/",
+                    "fileReplacements": [
+                        {
+                            "replace": "projects/demo/src/environments/environment.ts",
+                            "with": "projects/demo/src/environments/environment.prod.ts"
+                        }
+                    ],
+                    "optimization": true,
+                    "outputHashing": "all",
+                    "sourceMap": false,
+                    "namedChunks": false,
+                    "aot": true,
+                    "extractLicenses": true,
+                    "vendorChunk": false,
+                    "buildOptimizer": true,
```

Comment:
This line is copy-pasted from `production`-configuration.
I need to override only `baseHref` option and all other options from `production`.

Does anybody know a better way ?

Reply:
`Does anybody know a better way ?`

`nx build -c github, production`

Reply:
https://angular.io/guide/workspace-config
<img width="835" alt="image" src="https://user-images.githubusercontent.com/12021443/228836216-31890b80-7ad0-43a1-a453-10676f37b1cd.png">

Reply:
I will create issue to do it after upgrading of Nx.

Thanks!!

Reply:
* https://github.com/Tinkoff/maskito/issues/241